README.md

```diff
@@ -1 +1,61 @@
-# "Microverse Code Review" project
\ No newline at end of file
+# "Microverse Code Review" project
+
+> This is "Microverse Code Review" project where a student play a role of Code Reviewer.
+
+![screenshot](./images/MicroverseReview.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML/CSS
+- GitHub/VSCode
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/SaidRasinlic/MicroverseProjectReview/feature-branch/index.html)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- GitHub/VSCode/Linters
+
+### Setup
+
+- gh repo clone SaidRasinlic/MicroverseProjectReview
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@GitHub/SaidRasinlic](https://twitter.com/SaidRasinlic)
+- Twitter: [@Twitter/SaidRasinlic](https://twitter.com/SaidRasinlic)
+- LinkedIn: [LinkedIn/SaidRasinlic](https://www.linkedin.com/in/saidrasinlic)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] Nice review @SaidRasinlic 👍  but you can be more explanatory in your review by giving reasons why the change is required example `Kindly walk through your code and remove any unused links, this will make your codebase more professional`

Reply:
Great job. All changes implemented. 👍🏼